src/Features/CSharp/Portable/Wrapping/InitializerExpression/AbstractCSharpInitializerExpressionWrapper.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Microsoft.CodeAnalysis.CSharp.Indentation;
+using Microsoft.CodeAnalysis.Indentation;
+using Microsoft.CodeAnalysis.Wrapping.InitializerExpression;
+
+namespace Microsoft.CodeAnalysis.CSharp.Wrapping.InitializerExpression
+{
+    internal abstract class AbstractCSharpInitializerExpressionWrapper<TListSyntax, TListItemSyntax>
+        : AbstractInitializerExpression<TListSyntax, TListItemSyntax>
+        where TListSyntax : SyntaxNode
+        where TListItemSyntax : SyntaxNode
+    {
+        protected override string Indent_all_items => FeaturesResources.Indent_all_arguments;
```

Comment:
"arguments"?

Reply:
Whoops, I thought I changed that. I was going to look at removing these sub labels too unless you think they would be better left in. But, as of now each refactoring only has one option and that seems pointless.